### DISCUSSION TITLE
how to pen avatar camera ?

### DISCUSSION THREAD

**Discussion by nhatdn** [mozilla/hubs#5728] [General Questions & Discussion] [answered]
i want to open functional avatar camera but i dont know config it anywhere.. please helpe me, thanks for all.
![image](https://user-images.githubusercontent.com/85542339/191880001-8a9b1438-99c7-4ac3-87f5-e6814b09577a.png)

**Comment by matthewbcool** [comment, marked as answer]
To use the avatar camera, the avatar must have a video texture target. Not all avatars have this feature. You can build this by creating a custom avatar. See https://github.com/MozillaReality/hubs-avatar-pipelines/blob/master/Blender/AvatarBot/readme.md and https://www.youtube.com/watch?v=qBvZhh6KVcg for more details.

**Reply by nhatdn** [reply]
thanks you so much.
